tests/test_pytest_memray.py

```diff
@@ -271,11 +271,16 @@ def test_foo():
     )
 
     output = result.stdout.str()
-    mock.assert_called_once_with(
```

Comment:
Why did this need to change? Is this a holdover from an earlier iteration, perhaps? It seems to me like this old assertion should work fine with the currently proposed implementation.